docs/replica_bootstrap.rst

```diff
@@ -94,4 +95,11 @@ A special ``no_master`` parameter, if defined, allows Patroni to call the replic
 running master or replicas. In that case, an empty string will be passed in a connection string. This is useful for
 restoring the formerly running cluster from the binary backup.
 
+A ``basebackup`` method is a special case: it will be used if ``create_replica_method`` is empty, although it is possible
```

Comment:
I think we should also mention here `clonefrom` tag.

Patroni will prefer to use any existing replica tagged with clonefrom to take basebackup from it rather than from leader.

Reply:
Good point